### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by zaidmohd**
- Add Key Vault instructions to validate certificate from key vault
- Update K3s icon screenshots
- VSCode screenshots

### CODE REVIEW

File: docs/azure_jumpstart_arcbox/DevOps/_index.md [hunk 1]
```diff
@@ -618,15 +650,15 @@ Optionally, you can explore additional GitOps and RBAC scenarios in a manual fas
 
       Result should look like that:
```
Comment (dkirby-ms) [inline docs/azure_jumpstart_arcbox/DevOps/_index.md:651]:
```suggestion
      The result should look like this:
```

File: docs/azure_jumpstart_arcbox/DevOps/_index.md [hunk 2]
```diff
@@ -618,15 +650,15 @@ Optionally, you can explore additional GitOps and RBAC scenarios in a manual fas
...
   
-      ![Screenshot showing Hello-Arc app and shell](./k3s_gitops09.png)
+      ![Screenshot showing Hello-Arc app and shell](./k3s_gitops10.png)
   
     - In your fork of the “Azure Arc Jumpstart Apps” GitHub repository, open the _`hello_arc.yaml`_ file (_`/hello-arc/yaml/hello_arc.yaml`_). Change the replica to 2 and text under the “MESSAGE” section and commit the change.
 
-      ![Screenshot showing hello-arc repo](./k3s_gitops10.png)
+      ![Screenshot showing hello-arc repo](./k3s_gitops11.png)
 
     - Upon committing the changes, notice how the Kubernetes pods rolling upgrade will begin. Once the pods are up & running, refresh the browser, the new “Hello Arc” application version window will show the new message, showing the rolling upgrade is completed and the GitOps flow is successful.
```
Comment (dkirby-ms) [inline docs/azure_jumpstart_arcbox/DevOps/_index.md:659]:
```suggestion
    - Upon committing the changes, notice how the Kubernetes pods rolling upgrade will begin. Once the pods are up and running, refresh the browser, the new “Hello Arc” application version window will show the new message, showing the rolling upgrade is completed and the GitOps flow is successful.
```

**Review by dkirby-ms — CHANGES_REQUESTED** [review]
* Small language suggestions
* Why the "book" icon for the Hello-Arc app? Seems like that would be better suited for the bookstore
* Your user Object ID is visible in the Key vault permissions screenshot.

**Comment by zaidmohd** [comment]
> * Your user Object ID is visible in the Key vault permissions screenshot.
Done

**Pull request merged**